cluster/BUILD

```diff
@@ -46,3 +46,31 @@ pkg_tar(
         "//cluster/saltbase:salt-manifests",
     ],
 )
+
+# These tests just verify that bash can interpret the file.
+sh_test(
+    name = "common_test",
+    srcs = ["common.sh"],
+    data = [
+        ":all-srcs",
+        "//hack/lib:all-srcs",
```

Comment:
we actually have a `sh_library` in `hack/lib`, so `//hack/lib` is probably a more correct dep here.

Reply:
Done.